File: challenges/2024-02-08_00-appealing-skewer/js/challenge.js
export function calculateValidCombinations(ingredients) {
    // write your code here -----------------------------
    
    // --------------------------------------------------
}

process.env.npm_lifecycle_event === "start" && (_ => {
    const ingredients = ["🥔", "🥩", "🥩"];
    const expected = 2;
    const actual = calculateValidCombinations(ingredients);

    if (expected === actual)
        console.log(`SUCCESS | expected: ${expected} | got: ${actual}`);
    else
        console.log(`FAILED | expected: ${expected} | got: ${actual}`);
})();

File: challenges/2024-02-08_00-appealing-skewer/go/challenge.go
package main

import "fmt"

func calculateValidCombinations(ingredients []string) int {
	// write your code here -----------------------------

	// --------------------------------------------------
}

func main() {
	ingredients := []string{"🥔", "🥩", "🥩"}
	expected := 2
	actual := calculateValidCombinations(ingredients)

	if expected == actual {
		fmt.Printf("SUCCESS | expected: %d | got: %d", expected, actual)
	} else {
		fmt.Printf("FAILED | expected: %d | got: %d", expected, actual)
	}
}
